app/controllers/companies_controller.rb

```diff
@@ -6,6 +6,10 @@ class CompaniesController < ApplicationController
   end
 
   def dashboard
+    add_breadcrumb "Dashboard"
```

Comment:
give path for each breadcrumb 
gem will handle the rest

Reply:
In dashboard action, we should give the path of dashboard?